listenbrainz/webserver/static/js/src/utils.tsx

```diff
@@ -231,44 +231,54 @@ const formatWSMessageToListen = (wsMsg: any): Listen | null => {
 };
 
 // recieves or unix epoch timestamp int or ISO datetime string
-const preciseTimestamp = (listened_at: number | string): string => {
+const preciseTimestamp = (
+  listened_at: number | string,
+  displaySetting?: "timeAgo" | "includeYear" | "excludeYear"
```

Comment:
Updated this function so we can explicitly choose a setting to display timestamps (was needed to show timestamp in modal).